bqn-symbols-doc.el

```diff
@@ -3025,23 +2921,22 @@ at 9 ↕ \"short\"
      1 0 1 0 1 0
      0 0 0 0 0 0
      1 0 1 0 1 0
-                 ┘"]
+                 ┘"]))
 
-      ;; ================================================
-      ;; Group
-"⊔"
+;;;;; Group
+                   (("⊔" .
```

Comment:
@phikal why are you using two opening parens here, when in the other similar lines like 3044 you use only one?